### DISCUSSION TITLE
Add Infinity Wallet Connector

### DISCUSSION THREAD

**Discussion by infinitys2025** [wagmi-dev/references#115] [General]
Hello, we have been approached by several projects that are currently using wagmi for various wallet providers. However at the moment projects using wagmi are unable to support Infinity Wallet Web3 Browser connecting to its fullest.

Would it be possible for us to create a PR to add the Infinity Wallet Web3 Browser connector, to enable current partners and many other DApps to be able to support connecting.

**Comment by tmm** [comment]
Feel free to kick off a new [Connector Request](https://github.com/wagmi-dev/references/discussions/new?category=connector-request). Please note that not all Connector Requests are accepted. See the [Connector Guidelines](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#connectors) for more info.
